longevity_test.py

```diff
@@ -157,6 +157,10 @@ def test_custom_time(self):
             else:
                 params = {'keyspace_num': keyspace_num, 'stress_cmd': stress_cmd}
                 self._run_all_stress_cmds(stress_queue, params)
+        fullscan = self.params.get('fullscan', default=False)
+        if fullscan:
+            fullscan_interval = self.params.get('fullscan_interval', default=1)
+            self.run_fullscan_thread(timeout=fullscan_interval)
```

Comment:
can't we use one param instead of two?
even something like:
run_fullscan: 'table_name', 'interval_in_mins'
e.g.
run_fullscan: 'ks1.stabdrad1', 30
maybe even supporting random by using a special term:
run_fullscan: 'Random', 30

Reply:
i can mistaken, but yaml not support format
run_fullscan: 'ks1.standart1', 30

we can use next
run_fullscan: 'ks1.standart1, 30'

Reply:
Resolved!